common4j/src/main/com/microsoft/identity/common/java/telemetry/events/ApiStartEvent.java

```diff
@@ -182,6 +201,11 @@ public ApiStartEvent putValidationStatus(@NonNull final String validationStatus)
         return this;
     }
 
+    public ApiStartEvent putWorkPlaceJoined(final boolean isWorkPlaceJoined) {
```

Comment:
This is fine for now, but ultimately we want some of these things to probably move out of API Event